core/src/stored/stored_conf.cc

```diff
@@ -84,7 +84,7 @@ static ResourceItem store_items[] = {
   {"BackendDirectory", CFG_TYPE_STR_VECTOR_OF_DIRS, ITEM(res_store, backend_directories), 0,
       CFG_ITEM_DEFAULT | CFG_ITEM_PLATFORM_SPECIFIC, PATH_BAREOS_BACKENDDIR, NULL, NULL},
 #endif
-  {"JustInTimeReservation", CFG_TYPE_BOOL, ITEM(res_store, just_in_time_reservation), 0, CFG_ITEM_DEFAULT, "No", "24.0.0-",
+  {"JustInTimeReservation", CFG_TYPE_BOOL, ITEM(res_store, just_in_time_reservation), 0, CFG_ITEM_DEFAULT, "Yes", "23.1.0-",
```

Comment:
I have a doubt for 23.1.0 that we want to change the default behavior of the release.
With 23.1.x we  normally introduce a new technical feature jit in that case, but we certainly don't want to change the default, even if we know how good it can be and increase the global performance. This may be discussed around with the other devs.
```suggestion
  {"JustInTimeReservation", CFG_TYPE_BOOL, ITEM(res_store, just_in_time_reservation), 0, CFG_ITEM_DEFAULT, "No", "23.1.0-",
```

Reply:
So I cleaned it up.